CONTRIBUTION.md

```diff
@@ -0,0 +1,161 @@
+Table of Contents
+=================
+* [Code of Conduct of Eclipse Che documentation project](#code-of-conduct-of-eclipse-che-documentation-project)
+  * [Our Pledge](#our-pledge)
+  * [Our Standards](#our-standards)
+  * [Attribution](#attribution)
+* [Pull Request Process](#pull-request-process)
+   * [Release branches](#release-branches)
+* [Creating documentation](#creating-documentation)
+   * [AsciiDoc Formatting](#asciidoc-formatting)
+      * [Using test-adoc for AsciiDoc syntax verification](#using-test-adoc-for-asciidoc-syntax-verification)
+   * [Documentation Structure](#documentation-structure)
+      * [Using newdoc to create templates for new documents](#using-newdoc-to-create-templates-for-new-documents)
+   * [Writing documentation](#writing-documentation)
+      * [Style](#style)
+      * [Using vale to check the style](#using-vale-to-check-the-style)
+      * [Adding Images](#adding-images)
+   * [Building Documentation](#building-documentation)
+   * [Publishing documentation](#publishing-documentation)
+* [Getting Support](#getting-support)
+
+
+## Code of Conduct of Eclipse Che documentation project
+
+### Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience,
```

Comment:
"...we as contributors and maintainers pledge to make participation..."

"Make" instead of "making", basically. :)